File: apis/src/main/java/com/beat/apis/performance/application/dto/create/ScheduleRequest.java
package com.beat.apis.performance.application.dto.create;

import java.time.LocalDateTime;

import com.beat.apis.schedule.application.dto.ScheduleNumberType;

public record ScheduleRequest(
	LocalDateTime performanceDate,
	int totalTicketCount,
	ScheduleNumberType scheduleNumber
) {
}

File: apis/src/main/kotlin/com/beat/apis/performance/api/request/ScheduleRequest.kt
package com.beat.apis.performance.api.request

import com.beat.apis.schedule.api.type.ScheduleNumberType
import java.time.LocalDateTime

data class ScheduleRequest(
    val performanceDate: LocalDateTime,
    val totalTicketCount: Int,
    val scheduleNumber: ScheduleNumberType,
)
